### PR TITLE
Add cookbook for mkcert usage

### PR DESCRIPTION
**Pull request by pyrech**
This new cookbook explain how to use and setup mkcert and propose an Invoke task to ease user interactions with mkcert.

### CODE REVIEW

**Review by welcoMattic — APPROVED** [review]
:heart_eyes:

File: README.md [hunk 1]
```diff
@@ -643,6 +643,77 @@ environment variables you may script this with Invoke.
...
+
+<summary>Read the cookbook</summary>
+
+First, you need to remove the existing self-signed certificates and ignore the
+directory from git:
+
+```bash
+git rm infrastructure/docker/services/router/etc/ssl/certs/*.pem
+touch infrastructure/docker/services/router/etc/ssl/certs/.gitkeep
+echo '/infrastructure/docker/services/router/etc/ssl/certs/*.pem' | tee -a .gitignore
```
Comment (lyrixx) [inline README.md:658]:
```suggestion
echo '/infrastructure/docker/services/router/etc/ssl/certs/*.pem' >> .gitignore
```

File: README.md [hunk 2]
```diff
@@ -643,6 +643,77 @@ environment variables you may script this with Invoke.
...
+to make the application working in HTTPS out of the box.
+
+We suggest to use [`mkcert`](https://github.com/FiloSottile/mkcert#installation) to generate and sign your certificates.
+
+If you never installed CA root from mkcert before, you can do it by running `mkcert -install`. Now you can run the following
+command to generate the SSL certificates for the project:
+
+```bash
+inv generate-certificates
+```
```
Comment (lyrixx) [inline README.md:714]: There is an issue with triple backquote. You can indent `` ```bash `` block content. I'll be easier

**Review by lyrixx — COMMENTED** [review]
Top 👍🏼

**Review by lyrixx — APPROVED** [review]

**Pull request merged**